setup.py

```diff
@@ -105,9 +105,9 @@
     url="https://github.com/recommenders-team/recommenders",
     project_urls={
         "Documentation": "https://microsoft-recommenders.readthedocs.io/en/stable/",
-        "Wiki": "https://github.com/microsoft/recommenders/wiki",
+        "Wiki": "https://github.com/recommenders-team/recommenders/wiki",
     },
-    author="RecoDev Team at Microsoft",
+    author="Recommenders contributors",
     author_email="RecoDevTeam@service.microsoft.com",
```

Comment:
Email too is outdated.

Reply:
I have sent an email to Ibrahim to see which one we can add

Reply:
Tracked here: https://github.com/recommenders-team/recommenders/issues/1997